src/bindings/python/wheel/setup.py

```diff
@@ -426,6 +449,27 @@ def copy_package_data(self, src_dirs):
             packages.append(WHEEL_LIBS_PACKAGE)
             package_data.update({WHEEL_LIBS_PACKAGE: ["*"]})
 
+    def copy_package_data(self, src_dirs):
+        """Collect package data files from preinstalled dirs and put to the subpackage."""
+        package_dir = os.path.join(PACKAGE_DIR, WHEEL_PACKAGE_DIR)
+        os.makedirs(package_dir, exist_ok=True)
+
+        replacements = {
+            "{CMAKE_CURRENT_LIST_DIR}/../../": "{CMAKE_CURRENT_LIST_DIR}/../",
```

Comment:
could you please write comment for each replacement ? for future maintenance by other people

Reply:
added